crates/dbus/src/server.rs

```diff
@@ -105,14 +105,29 @@ impl Handler {
             app_icon,
             summary,
             body,
-            hints,
+            hints: hints.clone(),
             actions,
             expire_timeout,
             created_at,
             is_read: false,
         };
 
-        self.sender.send(Action::Show(notification.into())).unwrap();
+        if let Some(schedule) = &hints.schedule {
```

Comment:
Use this to avoid cloning the hints above.
```suggestion
        if let Some(schedule) = &notification.hints.schedule {
```